cmd/run/run.go

```diff
@@ -175,14 +175,11 @@ type ProfilerConfig struct {
 	Addr    string
 }
 
-type OpenTelemetryMetricsConfig struct {
-	Endpoint string
-	Protocol string
-}
-
-// OpenTelemetryConfig defines configurations for OpenTelemetry telemetry settings.
-type OpenTelemetryConfig struct {
-	OpenTelemetryMetricsConfig `mapstructure:"metrics"`
+// MetricConfig defines configurations for serving custom metrics from OpenFGA.
+type MetricConfig struct {
+	Enabled             bool
+	Addr                string
+	EnableRPCHistograms bool
```

Comment:
🤔 why have this as configurable?

Reply:
Histograms with large cardinality can be quite costly, and so I felt it was a good flag to have.  The cardinality of the metrics collected for RPC request duration isn't excessive necessarily, but if you have no need for RPC histograms then this cuts down on the impact of the metrics reported by OpenFGA. 

We can omit it in favor of enabling RPC histograms if `metrics.enabled=true` (by default). If others feel that is appropriate I can make that change 👍